var/spack/repos/builtin/packages/py-abipy/package.py

```diff
@@ -0,0 +1,79 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyAbipy(PythonPackage):
+    """Set of python modules and scripts to analyze the results of
+    ABINIT computations."""
+
+    homepage = "https://github.com/gmatteo/abipy"
+
+    version('master', git="https://github.com/gmatteo/abipy.git", branch='master')
```

Comment:
@adamjstewart 

It took longer than expected but I've finally released abipy 0.2 on PyPi.
v0.2 version requires:

    * pymatgen>=4.7.2  
    * numpy>=1.9 (pymatgen-v4.7.2 requirement)
    * seaborn (abipy requirement added in v0.2)

I've also moved the abipy repo to https://github.com/abinit/abipy
I would say that now we  can finally start to integrate AbiPy with Spack

FYI: 
I've done some tests with spack and abinit-v8.2.2 and I had problems with hdf5+netcdf4+openmpi when Abinit tries to perform very simple IO operations with hdf5.

The code crashes inside the MPI layer, more specifically inside MPI_COMM_DUP.
I can reproduce the error on three different machines we use for the Abinit test farm.
I will re-investigate the problem using the last spack version because I've seen recent commits related to MPI and hdf5.
Note that the majority of the post-processing tools in AbiPy require netcdf files and this is
the reason why I would like  to make hdf5+mpi the default variant.